### PR TITLE
Copy "download_jobs" Jenkinsfile from test-infra repository

### PR DESCRIPTION
**Pull request by mkowalski**
/cc @osherdp 
/cc @adriengentil

### CODE REVIEW

File: Jenkinsfile.download_logs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        stage('Cluster-logs') {
+            steps {
+                sh "make download_cluster_logs"
+            }
+        }
+
+        stage('Create tickets') {
+            steps {
+                sh "rm -rf assisted-installer-deployment"
+                sh "git clone https://github.com/openshift-assisted/assisted-installer-deployment --branch ${ASSISTED_INSTALLER_DEPLOYMENT_BRANCH}"
```
Comment (osherdp) [inline Jenkinsfile.download_logs:42]: You'll need to reverse that
Close test-infra and use assisted-installer-deployment

**Review by osherdp — CHANGES_REQUESTED** [review]

File: Jenkinsfile.download_logs (hunk 1 above)
Comment (osherdp) [reply Jenkinsfile.download_logs:48]: Sorry **clone

Comment (mkowalski) [reply Jenkinsfile.download_logs:49]: Correct, I did a clone of test-infra up in Init. In here I want to keep this clone to have an option to use job definition from master, but python scripts from a selected branch. What do you think?

File: Jenkinsfile.download_logs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    }
+    options {
+      timeout(time: 90, unit: 'MINUTES')
+    }
+
+    stages {
+        stage('Init') {
+            steps {
+                sh "rm -rf assisted-test-infra"
+                sh "git clone https://github.com/openshift-assisted/assisted-test-infra --branch ${ASSISTED_TEST_INFRA_BRANCH}"
```
Comment (osherdp) [inline Jenkinsfile.download_logs:30]: You'll need to fix this to openshift instead of openshift-assisted

File: Jenkinsfile.download_logs (hunk 1 above)
Comment (osherdp) [reply Jenkinsfile.download_logs:49]: Sounds good

**Review by osherdp — CHANGES_REQUESTED** [review]

File: Jenkinsfile.download_logs (hunk 2 above)
Comment (mkowalski) [reply Jenkinsfile.download_logs]: Of course, copy-paste mistake

**Review by osherdp — APPROVED** [review]

**Pull request merged**